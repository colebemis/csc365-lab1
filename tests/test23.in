RAW: teacher=HANTZ bus=51
Q