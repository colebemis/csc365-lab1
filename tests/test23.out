Enter query: 701dd7c11905a4c0e1a9; 3.12; 4; HANTZ, JED; 51
Enter query: 